.eslintrc.json

```diff
@@ -0,0 +1,25 @@
+{
```

Comment:
- [ ] Please remove all unnecessary files such as .DS_Store to keep your project clean.